feat(xo-web/new VM): warn if pool is not setup for secure boot (#7562)
vatesfr/xen-orchestra: CHANGELOG.unreleased.md

> Users must be able to say: “Nice enhancement, I'm eager to test it”

> - [VM] At VM creation, warn if secure boot is on but pool is not setup to (PR [#7562](https://github.com/vatesfr/xen-orchestra/pull/7562))

Reviewer: ```suggestion
- [VM] At VM creation, warn if secure boot is on but pool is not setup for UEFI Secure Boot (PR [#7562](https://github.com/vatesfr/xen-orchestra/pull/7562))
```

> 

Reviewer 2: ```suggestion
- [VM] At VM creation, warn if secure boot is on but pool is not setup for UEFI Secure Boot [#7500](https://github.com/vatesfr/xen-orchestra/issues/7500) (PR [#7562](https://github.com/vatesfr/xen-orchestra/pull/7562))

```

- @vates/task minor
- @xen-orchestra/proxy minor
- xo-server patch

> - xo-web minor

Reviewer 2: ```suggestion
- xo-server minor
- xo-web minor
```

- @xen-orchestra/mixins minor
- xen-api patch
- xo-cli patch

> - @vates/task minor
> - @xen-orchestra/proxy minor

Reviewer 2: ```suggestion
```
